go/tools/release-notes/release_notes.go

```diff
@@ -499,8 +499,11 @@ func main() {
 	pflag.StringVarP(&to, "to", "t", to, "to sha/tag/branch")
 	pflag.StringVarP(&versionName, "version", "v", "", "name of the version (has to be the following format: v11.0.0)")
 	pflag.StringVarP(&summaryFile, "summary", "s", "", "readme file on which there is a summary of the release")
+	pflag.IntVar(&numberOfThreads, "threads", numberOfThreads, "Define the number of threads used to fetch data from GitHub's API. Lower this number if you hit request limit errors.")
 	pflag.Parse()
 
+	log.Println(numberOfThreads)
+	os.Exit(1)
```

Comment:
I don't think we want either of these two lines, but definitely not the second one.

Reply:
Fixed via [62ad1cb](https://github.com/vitessio/vitess/pull/13278/commits/62ad1cb5f8c8478ef1a105da449d1ea3523b34a4), thank you!